src/main/java/webserver/RequestController.java

```diff
@@ -21,22 +22,36 @@ public static RequestController getRequestController() {
         return RequestControllerHelper.INSTANCE;
     }
 
-    public boolean isMappingView(Request request) {
+    public Response process(Request request) throws IOException {
+        //데이터 조회
         if (request.getMethod().equals("GET")) {
-            return !request.isParam();
+            // 동적 데이터 조회
+            if (request.isParam()) {
+                //Todo: 나중에 동적 데이터 조회 필요하면 만들자, 현재는 없는 요청이라서 404 에러페이지 보내기
+                return new Response(StatusCode.NOT_FOUND, "/error.html", request.getAccept());
+            } else { //정적 데이터 조회
+                logger.debug("request static data : {}", request.getPath());
+                return new Response(StatusCode.OK, request.getPath(), request.getAccept());
+            }
+        } else {
```

Comment:
```suggestion
        } else if(request.getMethod().equals("POST") {
```
현재 코드는 PUT이나 DELETE 같은 다른 요청도 POST로 처리가 되겠네요.